google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -27,7 +35,44 @@ public static CloudSdkService getInstance() {
     return ServiceManager.getService(CloudSdkService.class);
   }
 
+  @Nullable
   public abstract String getCloudSdkHomePath();
 
   public abstract void setCloudSdkHomePath(String path);
+
```

Comment:
And if I interpreted you correctly, how about we leave it in the service and open an enhancement against the common lib to expose this functionality for us.

I'll think about a better name for the service since its doing a bunch of appengine standard'y stuff now.